Tasks/Common/azure-arm-rest/azure-arm-mysql.ts

```diff
@@ -100,7 +100,7 @@ export class FirewallRules {
                 deferred.reject(new azureServiceClient.ApiResult(azureServiceClient.ToError(response)));
             }
             else if(statusCode === 202){
-                this._recursiveGetCall(resourceGroupName, serverName, firewallRuleName, 3, 0).then((response) => {
+                this._recursiveGetCall(resourceGroupName, serverName, firewallRuleName, 5, 0).then((response) => {
```

Comment:
instead of making it a constant. Can we make it a secondary variable so that user can customize the time, he need to retry?

Reply:
This can also help in checking the L2 by changing the variable in RD json.

Reply:
This is good idea but no customer has reported any issue due to this . Once they will ask We will add this one .